ipatests/test_integration/test_external_ca.py

```diff
@@ -279,3 +280,71 @@ def test_install_external_ca(self):
         # Install new cert
         self.master.run_command([paths.IPA_CACERT_MANAGE, 'install',
                                  root_ca_fname])
+
+
+class TestMultipleExternalCA(IntegrationTest):
+    """Setup externally signed ca1
+     install ipa-server with with externally signed ca1
+     Setup externally signed ca2 and renew ipa-server with
+     externally signed ca2 and check the difference in certificate"""
+
+    def test_master_install_ca1(self):
+
+        install_server_external_ca_step1(self.master)
+        # Sign CA, transport it to the host and get ipa a root ca paths.
+        root_ca_fname1 = tempfile.mkdtemp(suffix='root_ca.crt', dir=paths.TMP)
+        ipa_ca_fname1 = tempfile.mkdtemp(suffix='ipa_ca.crt', dir=paths.TMP)
+
+        # Get IPA CSR as bytes
+        ipa_csr = self.master.get_file_contents(paths.ROOT_IPA_CSR,)
+
+        external_ca = ExternalCA()
+        # Create root CA
+        root_ca = external_ca.create_ca(cn='RootCA1')
+        # Sign CSR
+        ipa_ca = external_ca.sign_csr(ipa_csr)
+
+        # Transport certificates (string > file) to master
+        self.master.put_file_contents(root_ca_fname1, root_ca)
+        self.master.put_file_contents(ipa_ca_fname1, ipa_ca)
+
+        # Step 2 of ipa-server-install.
+        install_server_external_ca_step2(self.master, ipa_ca_fname1,
+                                         root_ca_fname1)
+
+        cert_nick = "caSigningCert cert-pki-ca"
+        cert_out1 = self.master.run_command(['certutil', '-L', '-d',
+                                             paths.PKI_TOMCAT_ALIAS_DIR,
+                                             '-n', cert_nick]).stdout_text
+        assert "CN=RootCA1" in cert_out1
```

Comment:
This block is a bit hard to read because the function arguments are heavily indented. The ``.stdout_text`` attribute access is easy to mess, too. Other test codes commonly assigns the output of ``run_command`` to a name ``result`` and then checks its ``stdout_text`` property.

```
        result = self.master.run_command([
            'certutil', '-L', '-d', paths.PKI_TOMCAT_ALIAS_DIR,
            '-n', "caSigningCert cert-pki-ca"
        ])
        assert "CN=RootCA1" in result.stdout_text
```